Workout/SwimmingWorkout.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SwimmingWorkout.swift
+//  Workout
+//
+//  Created by Marco Boschi on 04/03/2017.
+//  Copyright © 2017 Marco Boschi. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import MBLibrary
+
+class SwimmingWorkout: Workout {
+	
+	override init(_ raw: HKWorkout, delegate del: WorkoutDelegate?) {
+		super.init(raw, delegate: del)
+		
+		if #available(iOS 10, *) {
+			self.addDetails()
+			self.addRequest(for: HKQuantityTypeIdentifier.distanceSwimming.getType()!, withUnit: .kilometer(), andTimeType: .ranged, searchingBy: .workout)
```

Comment:
I'll look into supporting different prefixes (kilo-, nothing and other supported by HealthKit) for different workout types.

Can you please check if a swimming workout is something like this:
![img_3177](https://cloud.githubusercontent.com/assets/20811558/23633007/54c4d520-02c4-11e7-900e-41214d38906b.PNG)
by going to Health app > Activity > Workouts > Show all data and selection a swimming workout, if the distance does not appear under *Workout Sample* we'll need to load distance samples by time.